docs/cloud/tutorials/http-redirector.md

```diff
@@ -0,0 +1,40 @@
+# Setup a HTTP redirection in Rahti
+
+It is very simple to setup a HTTP redirection in Rahti using a web server like nginx. We can redirect to basically any URL we want with very complex logics. For this tutorial, we will keep it simple and simply redirect to a different host, but keeping the path of the URL.
+
+## Procedure
+
+1. Deploy an NGINX Image. It is recommended to use `bitnami/nginx`
+
+    ![bitnami/nginx](../img/bitnami-nginx.png)
+
+1. Add a Route with the URL that you want to redirect. If you visit the URL, you should see the "nginx welcome page"
+
+    ![route](../img/route-create.png)
+
+1. Add a ConfigMap with a server redirection block. Go to **Resources > Config Maps**, click in **Create Config Map**. The **Name** will be later used as **Source** when mounting the ConfigMap. The **Key** will be the name of the file, and the **Value** the content of the file. 
+
+    ```nginx
+    #default.conf
+    server {
+            listen *:8000;
```

Comment:
```suggestion
            listen *:8080;
```
When deploying the `bitnami/nginx` image and created a route, the target port is `8080` and `8000`